docs/reference/builder.md

```diff
@@ -817,14 +817,14 @@ change them using `docker run --env <key>=<value>`.
 
 ADD has two forms:
 
-- `ADD <src>... <dest>`
-- `ADD ["<src>",... "<dest>"]` (this form is required for paths containing
+- `ADD [--chown <user>:<group>] <src>... <dest>`
```

Comment:
Sure about that syntax? Not working for me & @presto9292 see https://github.com/moby/moby/pull/34263#issuecomment-328034492

Reply:
Thanks @shouze ; you are correct that it requires an `=`; I will update this PR as it also has other edits required before being complete.